conda/meta.yaml

```diff
@@ -20,11 +20,12 @@ build:
 
 test:
   requires:
-    - ts-conda-build
+    - ts-conda-build =0.3
     - ts-simactuators
     - ts-dds
     - ts-idl {{ idl_version }}
     - ts-salobj {{ salobj_version }}
+    - black
```

Comment:
Do you need black here since it is provided by the metapackage?

Reply:
Nope so I removed it.